.eslintrc.yml

```diff
@@ -19,18 +19,18 @@ rules:
   indent: [error, 2, {FunctionDeclaration: {body: 1, parameters: 1}, FunctionExpression: {body: 1, parameters: 1}, CallExpression: {arguments: 1}, SwitchCase: 1, MemberExpression: 1}]
```

Comment:
```suggestion
  "@typescript-eslint/indent": [error, 2, {FunctionDeclaration: {body: 1, parameters: 1}, FunctionExpression: {body: 1, parameters: 1}, CallExpression: {arguments: 1}, SwitchCase: 1, MemberExpression: 1}]
```

Reply:
@DedaDev I believe we should disable all formatting rules and install prettier instead.